.directory

```diff
@@ -0,0 +1,6 @@
+[Dolphin]
```

Comment:
In the first commit you deleted the Dolphin config and here you are readding it again by accident. Please remove it again from the second commit.